drake/systems/controllers/InstantaneousQPController.h

```diff
@@ -0,0 +1,99 @@
+#ifndef INSTANTANEOUSQPCONTROLLER_H
+#define INSTANTANEOUSQPCONTROLLER_H
+
+#include <memory>
+#include "QPCommon.h"
+#include "drake/solvers/gurobiQP.h"
+#include "lcmtypes/drake/lcmt_qp_controller_input.hpp"
+
+#define USE_FASTQP 1
+#define GUROBI_OUTPUTFLAG 0
+#define GUROBI_METHOD 2
+#define GUROBI_PRESOLVE 0
+#define GUROBI_BARITERLIMIT 20
+#define GUROBI_BARHOMOGENEOUS 0
+#define GUROBI_BARCONVTOL (5e-4)
+
+
+class InstantaneousQPController {
```

Comment:
The NewQPControllerData struct and the methods that operated on it have now been merged into the new InstantaneousQPController class